### PR TITLE
[docs] Add recipe - using TestCafe with Gulp.js

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs

### CODE REVIEW

File: docs/articles/documentation/recipes/using-testcafe-with-gulp.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+layout: docs
+title: Using TestCafe with Gulp
+permalink: /documentation/recipes/using-testcafe-with-gulp.html
+---
+# Using TestCafe with Gulp
+
+If you are using Gulp to organize your build process,
+you can easily integrate TestCafe into the workflow.
+
+Suppose you have a GitHub project with Gulp.js set to run build tasks. To integrate TestCafe into your project, go through these steps.
```
Comment (kirovboris) [inline docs/articles/documentation/recipes/using-testcafe-with-gulp.md]: Is project on github a mandatory condition? Why?

File: docs/articles/documentation/recipes/using-testcafe-with-gulp.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+Run this task through the command line.
+
+```sh
+gulp run-testcafe-tests
+```
+
+Or create and run a dependent task.
+
+```js
+gulp.task('build', ['run-testcafe-tests'], () => {
```
Comment (kirovboris) [inline docs/articles/documentation/recipes/using-testcafe-with-gulp.md]: It looks strange. In general you build before tests. Probably, did you mean publish task or something else?

File: docs/articles/documentation/recipes/using-testcafe-with-gulp.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ gulp run-testcafe-tests
 Or create and run a dependent task.
 
 ```js
-gulp.task('build', ['run-testcafe-tests'], () => {
+gulp.task('publish', ['run-testcafe-tests'], () => {
```
Comment (AlexanderMoskovkin) [inline docs/articles/documentation/recipes/using-testcafe-with-gulp.md]: Maybe just `gulp.task('run-testcafe-tests')` ?
I suppose gulp users know what they should do with it

**Comment by VasilyStrelyaev** [comment]
@testcafe-build-bot \retest

**Comment by VasilyStrelyaev** [comment]
@testcafe-build-bot \retest

File: docs/articles/documentation/recipes/using-testcafe-with-gulp.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+This task runs tests from the `tests/test.js` file in Chrome and Firefox.
+
+For complete API Reference, see [the plugin page](https://github.com/DevExpress/gulp-testcafe#gulp-testcafe).
+
+## Step 3 - Run the TestCafe Task
+
+Run this task through the command line.
+
+```sh
+gulp run-testcafe-tests
```
Comment (churkin) [inline docs/articles/documentation/recipes/using-testcafe-with-gulp.md]: for this `gulp` should be installed globally

Comment (VasilyStrelyaev) [inline docs/articles/documentation/recipes/using-testcafe-with-gulp.md]: >Suppose you have a project with Gulp.js set to run build tasks.

We assume it is.

Comment (VasilyStrelyaev) [inline docs/articles/documentation/recipes/using-testcafe-with-gulp.md]: Or, am I wrong and people use `gulp` without installing it globally? \cc @DevExpress/testcafe-docs

Comment (AlexanderMoskovkin) [inline docs/articles/documentation/recipes/using-testcafe-with-gulp.md]: I don't think so

**Comment by AlexanderMoskovkin** [comment]
lgtm /cc @churkin

**Pull request merged**